spyder_terminal/config.py

```diff
@@ -19,7 +19,8 @@
      {
       'sound': True,
       'cursor_type': 0,
-      'shell': 'cmd' if WINDOWS else 'bash'
+      'shell': 'cmd' if WINDOWS else 'bash',
+      'buffer_lim': 1000
```

Comment:
Please change the name of this option to `buffer_limit` to be more explicit about it.